example_site/src/components/TemplateBrowser.js

```diff
@@ -102,20 +97,23 @@ function CollectionFilter({
   setSelectedCollection,
 }) {
   const filterNameColor = selectedCollection === collection ? blueColor : blackColor;
-  const filterFontWeight = selectedCollection === collection ? '800' : '400';
+  const filterFontWeight = selectedCollection === collection ? 'var(--fontWeightHeavy)' : 'var(--fontWeightRegular)';
 
   const onClick = (collection) => {
     setSelectedCollection(collection);
   };
 
   return (
-    // <div className="category-name">
-      <button className="filter-name" onClick={() => onClick(collection)}>
-        <div style={{'color': filterNameColor, 'font-weight': filterFontWeight}}>
-          {collection.name} 
-        </div>
-      </button>
-    // </div>
+    <button className="filter-name" onClick={() => onClick(collection)}>
```

Comment:
I missed this in the first review, but you don't need to pass `collection` in as a function argument here since it's the only in-scope collection. 

We use this format a lot when we're iterating through objects to ensure that the correct item is used in the function, but you can honestly just pass in `setSelectedCollection` like this:
`onClick={() => setSelectedCollection(collection)}` instead of wrapping it in another function